lib/ApiRequestor.php

```diff
@@ -145,16 +145,26 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
-        $curlVersion = curl_version();
+
+        $httplib = 'curl not installed';
```

Comment:
Should this also just be "unknown" like you have for `$ssllib`? Otherwise might look a little strange in our user agent header.